client/client.go

```diff
@@ -17,6 +17,8 @@ import (
 var (
 	defaultHost    = "unix:///var/run/pouchd.sock"
 	defaultTimeout = time.Second * 10
+	// defaultVersion is the version of the current stable API
+	defaultVersion = "0.2.0"
```

Comment:
This version should be compatible with Moby, so it should be v1.24, I think.

Reply:
agree